### PR TITLE
PORT base_url

### PR DESCRIPTION
**Pull request by bealdav**
Note:
In v10 methods which inherits of models.Model have `api.multi` default behavior (see last odoo days @ antonylesuisse) so I remove them and attach `api.model` to other methods.

### CODE REVIEW

File: base_url/models/base_url.py [hunk 1]
```diff
@@ -80,18 +79,19 @@ class AbstractUrl(models.AbstractModel):
         compute='_compute_redirect_url',
         comodel_name='url.url')
 
+    @api.model
     def _build_url_key(self):
         return slugify(self.record_id.with_context(
```
Comment (hparfr) [inline base_url/models/base_url.py]: Todo : refactor it.  record_id may exist or not. May be @api.somethingelse

**Review by hparfr — CHANGES_REQUESTED** [review]

**Comment by bealdav** [comment]
ping @hparfr fixed

**Comment by legalsylvain** [comment]
Hi @bealdav thanks ! 
2 things : could you 
-  split .py file to have one file per model ? 
- improve / fix manifest file. (not OCA module for instance, and description is quite poor).

regards.

**Review by hparfr — APPROVED** [review]

File: base_url/__manifest__.py [hunk 2]
```diff
@@ -6,11 +6,10 @@
...
     'name': 'Base Url',
-    'version': '8.0.0.1.0',
+    'version': '10.0.0.1.0',
     'category': 'tools',
     'license': 'AGPL-3',
     'summary': 'keep history of url for products & categories  ',
-
     'author': 'Akretion',
     'website': 'http://www.akretion.com/fr',
     # any module necessary for this one to work correctly
```
Comment (legalsylvain) [inline base_url/__manifest__.py:15]: should not depend on base ?

Comment (bealdav) [reply base_url/__manifest__.py:15]: good catch, fixed in v10. Thanks

**Pull request merged**